doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,982 @@
+
+  * :ref:`CONFIG_DESKTOP_USB_REMOTE_WAKEUP <config_desktop_app_options>` Kconfig option for :ref:`nrf_desktop_usb_state`.
+    The option enables the USB wakeup functionality in the application.
+    The option selects :kconfig:option:`CONFIG_USB_DEVICE_REMOTE_WAKEUP`.
+
+  * Application-specific Kconfig options to simplify the configuration.
+    Part of an application Kconfig configuration that is common for the selected HID device type is introduced as overlays for default Kconfig values.
+    See :ref:`nrf_desktop_porting_guide` for details.
+
+* Updated:
+
+  * The ``CONFIG_DESKTOP_BLE_SCANNING_ENABLE`` Kconfig option has been renamed to :ref:`CONFIG_DESKTOP_BLE_SCAN_ENABLE <config_desktop_app_options>`.
+  * The UUID16 values of GATT Human Interface Device Service (HIDS) and GATT Battery Service (BAS) have been moved from advertising data to scan response data.
+  * The :kconfig:option:`CONFIG_BT_GATT_SERVICE_CHANGED` Kconfig option is disabled on nRF Desktop dongles to reduce memory footprint.
+  * The :kconfig:option:`CONFIG_BT_ID_UNPAIR_MATCHING_BONDS` is now enabled by default.
+    This is done to pass the Fast Pair Validator's end-to-end integration tests and to improve the user experience during the erase advertising procedure.
+  * The :kconfig:option:`CONFIG_BT_ID_ALLOW_UNAUTH_OVERWRITE` Kconfig option is enabled by default for the HID peripherals.
+    This setting improves the user experience as it is no longer required to delete the bonding information from the old identity to pair using the new one.
+
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`peripheral_cgms` sample.
+
+* :ref:`ble_throughput` sample:
+
+  * Added terminal commands for selecting the role.
+  * Updated the ASCII art used for showing progress to feature the current Nordic Semiconductor logo.
+
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added:
+
+    * Bond removal functionality.
+    * Battery information to demonstrate the Fast Pair Battery Notification extension.
+    * TX power correction value to align the advertised TX power with the Fast Pair expectations.
+
+  * Updated:
+
+    * The sample now uses :ref:`bt_le_adv_prov_readme` to generate Bluetooth advertising and scan response data.
+    * The advertising pairing mode (:c:member:`bt_le_adv_prov_adv_state.pairing_mode`) is enabled only in the Fast Pair discoverable advertising mode.
+      The device also rejects normal Bluetooth LE pairing when not in the pairing mode.
+    * After the device reaches the maximum number of paired devices (set by :kconfig:option:`CONFIG_BT_MAX_PAIRED`), the device stops looking for new peers.
+      Therefore, the Fast Pair payload is no longer included in the advertising data.
```

Comment:
This is not valid. It should be:

` The device no longer advertises with the pairing mode (:c:member:bt_le_adv_prov_adv_state.pairing_mode) enabled, and only the Fast Pair not discoverable advertising with hide UI indication mode includes the Fast Pair payload.`